app/components/schedule.ts

```diff
@@ -120,8 +120,14 @@ export default class Schedule extends Component<ScheduleArgs> {
    */
   adjustMinTime(view: FullCalendarView, calendar: JQuery<HTMLElement>): void {
     if (isTesting || !(view.type === 'agendaDay' || view.type === 'timelineDay')) {return}
-
-    let minTime = '0:00:00';
+    let min_time = '24:00:00';
+    this.args.sessions.map(x => {
+      const z = String(x.startsAt.format('HH:mm:ss'));
+      if (min_time > z) {
+        min_time = z;
```

Comment:
https://stackoverflow.com/questions/1669190/find-the-min-max-element-of-an-array-in-javascript#6102340

Reply:
this is not working on string values